piet-direct2d/Cargo.toml

```diff
@@ -14,12 +14,9 @@ piet = { version = "0.2.0", path = "../piet" }
 utf16_lit = "1.0"
 
 wio = "0.2.2"
-
-[dependencies.winapi]
-version = "0.3.8"
-features = ["d2d1", "d2d1_1", "d2d1effects", "d3d11", "dxgi", "winnls"]
+winapi = { version = "0.3.8", features = ["d2d1", "d2d1_1", "d2d1effects", "d3d11", "dxgi", "winnls"] }
+dwrote = "0.11.0"
```

Comment:
I'm not happy with a dependency on dwrote, at least at the moment, as it has safety problems. Also we have a patchwork where we use different crates for different things. I appreciate the fact that using it is a lot less code, so am not going to block approval, but wonder what's the best way forward. One is to have an issue to remove the dwrote dep (by duplicating the functionality). Another possibility that occurs to me is to file an issue on dwrote, and see what happens.

Reply:
Agree this isn't the ideal solution; it's totally a matter of expediency. I'm happy to open an issue to remove this dep once com-rs has matured a little bit.

Reply:
One other thing about this: I think dwrote pulls in serde by default, so better to turn off default features.